include/config/overworld.h

```diff
@@ -9,6 +9,7 @@
 
 // PC settings
 #define OW_PC_PRESS_B               GEN_LATEST // In Gen4, pressing B when holding a Pokémon is equivalent to placing it. In Gen3, it gives the "You're holding a Pokémon!" error.
+#define OW_PC_JAPAN_WALDA_ICONS     FALSE // The Japanese release has the Cross, Bolt, and Plusle icons for Walda's wallpapers, but in the US release, they're blank
```

Comment:
I think it'd be nice to have this on by default, as not many people even know of Walda's wallpapers in the first place, and even less would bother to change this on their own.
```suggestion
#define OW_PC_JAPAN_WALDA_ICONS     TRUE       // In the US release of Emerald, the Cross, Bolt, and Plusle icons for Walda's wallpapers were left blank from the Japan release. Setting this to TRUE will restore them.
```
Besides that and the proposed comment change, looks good to me :)

Reply:
Just commited the changes! Describing things has never been my forte so thanks for including a better description